content/id/docs/concepts/services-networking/service-topology.md

```diff
@@ -0,0 +1,206 @@
+preferensi untuk menjaga layanan lalu lintas jaringan dalam zona yang sama, karena lalu lintas jaringan
+antar zona memiliki biaya yang dibebankan, sementara lalu lintas jaringan
+dalam zona yang sama tidak ada biaya. Kebutuhan umum lainnya termasuk kemampuan untuk merutekan 
+lalu lintas jaringan ke Pod lokal yang dikelola oleh sebuah DaemonSet, atau menjaga lalu lintas jaringan ke 
+Node yang terhubung ke _top-of-rack switch_ yang sama untuk mendapatkan 
+latensi terendah.
+
+
+## Menggunakan Topologi Service
+
+Jika klaster kamu mengaktifkan topologi layanan kamu dapat mengontrol rute lalu lintas jaringan Service
+dengan mengatur bagian `topologyKeys` pada spesifikasi Service. Bagian ini
+adalah daftar urutan label-label Node yang akan digunakan untuk mengurutkan _endpoint_
+saat mengakses Service ini. Lalu lintas jaringan akan diarahkan ke Node yang nilai
+label pertamanya cocok dengan nilai dari Node asal untuk label yang sama. Jika
+tidak ada _backend_ untuk Service pada Node yang sesuai, maka label kedua akan
+dipertimbangkan, dan seterusnya, sampai tidak ada label yang tersisa.
+
+Jika tidak ditemukan kecocokan, lalu lintas jaringan akan ditolak, sama seperti jika tidak ada
+sama sekali _backend_ untuk Service tersebut. Artinya, _endpoint_ dipilih 
+berdasarkan kunci topologi yang pertama yang tersedia pada _backend_. Jika dalam
+bagian ini ditentukan dan semua entri tidak memiliki _backend_ yang sesuai dengan
+topologi klien, maka Service tidak memiliki _backend_ untuk klien dan koneksi harus 
+digagalkan. Nilai khusus `" * "` dapat digunakan untuk mengartikan "topologi 
+apa saja". Nilai _catch-all_ ini, jika digunakan, maka hanya sebagai 
+nilai terakhir dalam daftar.
+
+Jika `topologyKeys` tidak ditentukan atau kosong, tidak ada batasan topologi 
+yang akan diterapkan.
+
+Seandainya sebuah klaster dengan Node yang dilabeli dengan nama _host_ , 
+nama zona, dan nama wilayah mereka, maka kamu dapat mengatur nilai 
+`topologyKeys` dari sebuah Service untuk mengarahkan lalu lintas jaringan seperti berikut ini.
+
+* Hanya ke _endpoint_ dalam Node yang sama, gagal jika tidak ada _endpoint_ pada
+  Node: `[" kubernetes.io/hostname "]`.
+* Lebih memilih ke _endpoint_ dalam Node yang sama, jika tidak ditemukan maka ke _endpoint_ pada
+  zona yang sama, diikuti oleh wilayah yang sama, dan selain itu gagal: 
+  `[" kubernetes.io/hostname ", "topology.kubernetes.io/zone", "topology.kubernetes.io/region"] `.
+  Ini mungkin berguna, misalnya, dalam kasus di mana lokalitas data sangat penting.
+* Lebih memilih ke _endpoint_ dalam zona yang sama, tetapi memilih _endpoint_ mana saja yang 
+  tersedia apabila tidak ada yang tersedia dalam zona ini:
+  `[" topology.kubernetes.io/zone "," * "]`.
+
+
+## Batasan
+
+* Topologi Service tidak kompatibel dengan `externalTrafficPolicy=Local`, dan
+   karena itu Service tidak dapat menggunakan kedua fitur ini sekaligus. Dimungkinkan untuk menggunakan
```

Comment:
nit: this adds extra spaces to the page, you can take a look https://deploy-preview-20229--kubernetes-io-master-staging.netlify.app/id/docs/concepts/services-networking/service-topology/#batasan

same with the lines below

Reply:
you can address this as follow up PR

Reply:
Ok... I will do it :)

Reply:
Upload in PR #20500